pkgs/development/tools/misc/fzf-make/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, makeBinaryWrapper
+, gnumake
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "fzf-make";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "kyu08";
+    repo = "fzf-make";
+    rev = "v${version}";
+    hash = "sha256-Rpz10sHu/pzy/WX1yj0N78gVaw0fO8D92TSdtkmL/MU=";
+  };
+
+  cargoHash = "sha256-vy1LJciZMwMo0gxoEtfgESYnUPRhbpnyvciRZHk1Oao=";
+
+  nativeBuildInputs = [
+    makeBinaryWrapper
+  ];
+
+  postInstall = ''
+    wrapProgram $out/bin/fzf-make \
+      --suffix PATH : ${lib.makeBinPath [ gnumake ]}
+  '';
+
+  meta = with lib; {
+    description = "Fuzzy finder for Makefile";
+    homepage = "https://github.com/kyu08/fzf-make";
+    changelog = "https://github.com/kyu08/fzf-make/releases/tag/${src.rev}";
+    license = licenses.mit;
+    maintainers = with maintainers; [ figsoda ];
+    mainProgram = "fzf-make";
```

Comment:
Does this not default to pname?

Reply:
It won't be in the future for `lib.getExe`: #246386

Reply:
We need something like a "nixpkgs news dashboard" for these things.

Reply:
We need something like a "nixpkgs news dashboard" for these things.